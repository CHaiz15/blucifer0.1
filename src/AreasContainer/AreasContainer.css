.areas-container {
  display: flex;
  flex-wrap: wrap;
  color: black;
  text-decoration: none;
}

.areas-title {
  text-align: center;
}
