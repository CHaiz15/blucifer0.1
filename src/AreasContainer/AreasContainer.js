import React from 'react';
import './AreasContainer.css'
import AreaCard from '../AreaCard/AreaCard.js'
import { Link } from 'react-router-dom';

const AreasContainer = ({ areas, addSelectedArea }) => {
  const appendedAreas = areas.map(area => 
    <Link className="area-card" to={`/nav/areas/${area.id}/listings`}>
      <AreaCard 
        addSelectedArea={addSelectedArea}
        id={area.id}
        areaName={area.name} 
        shortName={area.shortName} 
        about={area.about} 
        listings={area.listings}
      />
    </Link>
  )
  return(
    <section className="post-login">
      <h2 className="areas-title">Pick a neighborhood</h2>
      <section className="areas-container">
        {appendedAreas}
      </section>
    </section>
  )
}

export default AreasContainer;
