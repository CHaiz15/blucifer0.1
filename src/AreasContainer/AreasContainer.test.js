import React from 'react';
import { render } from '@testing-library/react';
import AreasContainer from './AreasContainer';
import { shallow } from 'enzyme'

describe('AreasContainer', () => {
  it('should be an instance of AreaContainer component', () => {
    const wrapper = shallow(
        <AreasContainer
            addSelectedArea={jest.fn()}
            areas={[]}
        />);
      expect(wrapper).toMatchSnapshot();
  });
})