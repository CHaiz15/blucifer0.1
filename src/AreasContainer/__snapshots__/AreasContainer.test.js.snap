// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AreasContainer should be an instance of AreaContainer component 1`] = `
<section
  className="post-login"
>
  <h2
    className="areas-title"
  >
    Pick a neighborhood
  </h2>
  <section
    className="areas-container"
  />
</section>
`;
